bitnami/redis/README.md

```diff
@@ -501,7 +501,7 @@ The command removes all the Kubernetes components associated with the chart and
 | -------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------- |
 | `useExternalDNS.enabled`               | Enable various syntax that would enable external-dns to work.  Note this requires a working installation of `external-dns` to be usable. | `false`                             |
 | `useExternalDNS.additionalAnnotations` | Extra annotations to be utilized when `external-dns` is enabled.                                                                         | `{}`                                |
-| `useExternalDNS.annotationKey`         | The annotation key utilized when `external-dns` is enabled.                                                                              | `external-dns.alpha.kubernetes.io/` |
+| `useExternalDNS.annotationKey`         | The annotation key utilized when `external-dns` is enabled. Setting this to `false` will disable annotations.                            | `external-dns.alpha.kubernetes.io/` |
```

Comment:
Please add this change here:
https://github.com/bitnami/charts/blob/ba935a6b9af28c814a244f628a787033abdd92ae/bitnami/redis/values.yaml#L1614

This README section is autogenerated from `values.yaml`

Reply:
Done!